example/lib/storybook/stories/card_rounded_story.dart

```diff
@@ -0,0 +1,55 @@
+import 'package:flutter/material.dart';
+import 'package:help_ukraine_widget/help_ukraine_widget.dart';
+import 'package:widgetbook/widgetbook.dart';
+
+///
```

Comment:
Let's fill this with some useful info:

```suggestion
/// Definition of available states and customizable parameters for [CardRounded].
```